### DISCUSSION TITLE
Does the Xoreos team still have twMax?

### DISCUSSION THREAD

**Discussion by ReindeerFloatilla** [xoreos/xoreos#597] [General]
There was an article posted to the Xoreos blog here mentioning a 3dsmax script called twMax for The Witcher.

https://xoreos.org/blog/2015/04/12/the-witcher-models-and-areas/

That script is now lost media the creator's website is offline. Does the Xoreos team still have it? If you do can you please send it to me so I can reupload it to moddb?

**Comment by DrMcCoy** [comment]
Yes, I still have Michael_DarkAngel's twMax 1.2.3.2 from 2009-02-03 here, which is according to what I can see on the Wayback Machine the latest version that was available on its website before it vanished.

I'm attaching it here:
[twmax-1.2.3.2.zip](https://github.com/xoreos/xoreos/files/13203395/twmax-1.2.3.2.zip)

**Comment by ReindeerFloatilla** [comment]
Thank you SO MUCH you are the best.
